client/packages/system/src/Report/api/hooks/document/useReports.ts

```diff
@@ -1,7 +1,16 @@
 import { useQuery, ReportContext } from '@openmsupply-client/common';
+import { ReportListParams } from '../../api';
 import { useReportApi } from '../utils/useReportApi';
 
-export const useReports = (context?: ReportContext, subContext?: string) => {
+export const useReports = ({
+  context,
+  subContext,
+  queryParams,
+}: {
+  context?: ReportContext;
+  subContext?: string;
+  queryParams?: ReportListParams;
```

Comment:
I've added the optional params so that we can choose to use urlQueryParams from the listview and not use the url params from the more standard usage. Required in order to get the sort and filter working correctly.

Reply:
+1 Had something similar with @CarlosNZ the other day. Think it also makes the hook easier to understand if you pass in the parameter explicitly, e.g. initially I was confused where hooks are getting their parameters from. Moreover, by having more params the hook can be used in more places, e.g. some of the program hooks can only be used in certain views and otherwise silently don't work if you use them at the wrong place.